conda_recipies/elasticsearch/meta.yaml

```diff
@@ -0,0 +1,14 @@
+package:
+    name: elasticsearch
+    version: "1.6.0"
+
+source:
+    git_rev: v1.6.0
+    git_url: https://github.com/elastic/elasticsearch.git
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - apache-maven
```

Comment:
Just a heads-up, it looks like the version tracked in https://github.com/conda/conda-recipes/blob/master/apache-maven/build.sh has JAVA_HOME issues and will not work on a modern Ubuntu box.